presto-benchto-benchmarks/src/main/resources/sql/presto/tpch/q1.sql

```diff
@@ -10,7 +10,7 @@ SELECT
   avg(l.discount)                                       AS avg_disc,
   count(*)                                              AS count_order
 FROM
-  "${database}"."${schema}"."lineitem" AS l
+  "${database}"."${schema}"."${prefix}lineitem" AS l
```

Comment:
On my side I want to invest in improvements when I have working system end-to-end. I'm close to this.